src/proposals/20250529/SparkUnichain_20250529.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.25;
+
+import { Ethereum } from "spark-address-registry/Ethereum.sol";
+
+import {
+    ControllerInstance,
+    ForeignControllerInit
+} from "spark-alm-controller/deploy/ForeignControllerInit.sol";
+import { RateLimitData } from "spark-alm-controller/src/RateLimitHelpers.sol";
```

Comment:
Isn't an empty line between imports missed?

I see the same in Arbitrum, but I guess it was missed there as well

Reply:
they are from the same module hence grouped together